### ISSUE TITLE
[ERROR] The builder requested using id = incremental cannot be found

### ISSUE THREAD

**Issue by Sam123ben** [khmarbaise/incremental-module-builder#8]
When tried to run your recommended approach getting the below error message:

[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.770 s
[INFO] Finished at: 2017-06-16T11:57:00+12:00
[INFO] Final Memory: 6M/121M
[INFO] ------------------------------------------------------------------------
[ERROR] The builder requested using id = incremental cannot be found -> [Help 1]

Hi, how I did was by running the maven mvn clean install for creating the jar file first and then tried running the multimodule project. Not sure did I missed anything???

1. Mvn clean install to create and install the jar
2. Then added the snippet you shared on ur forum in .mvn/extension.xml
3. Tried running the multimodule project using the -b incremental and then showed the issue

**Comment by khmarbaise** [comment]
It looks like you haven't correctly installed the extension in .mvn/extensions.xml cause there is no output which shows that the extension is active...which must be there...

**Comment by Sam123ben** [comment]
Ok so if I am not wrong what I did is I used the snippet you have mentioned in your guide in the extension.xml file under .mvn folder. Do I need to give any command specifically to install the extension???

I just used the group id, artifactory I'd, and the version in the extension file then ran mvn clean install and then the incremental builder command but that failed

**Comment by khmarbaise** [comment]
Ah..you need to change the version to 0.2.0 cause 0.3.0 is not yet on Maven Central..this can cause the issue...

**Comment by Sam123ben** [comment]
Yeah when I saw your forum and some other comments I did change that to the version you mentioned.

**Comment by Sam123ben** [comment]
Anyways let me try again tomorrow sometime and see if any of my trial and error method could work and if will update the same here too. Thanks
